### PR TITLE
Feature auth

### PR DESCRIPTION
**Pull request by bilenkomike**

### CODE REVIEW

File: src/pages/Login/Login.jsx [hunk 1]
```diff
@@ -1,25 +1,46 @@
...
+import { Typography } from "@mui/material";
 
 function Login() {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
 
+  const [emailValid, setEmailValid] = useState(false);
+  const [passwordValid, setPasswordValid] = useState(false);
+
+  const setEmailFieldValid = (validVal) => {
```
Comment (myroslavsuprun) [inline src/pages/Login/Login.jsx:13]: useState goes first, than useEffectt than additional functions

File: src/pages/Login/Login.jsx [hunk 2]
```diff
@@ -1,25 +1,46 @@
...
     <Container maxWidth="xs">
+      <Typography align="center" gutterBottom variant="h2">
+        Login
+      </Typography>
       <form onSubmit={submitHandler}>
         <FormInput
           label="Email:"
           id="email"
           change={setEmail}
           cont={email}
```
Comment (myroslavsuprun) [inline src/pages/Login/Login.jsx:41]: What is cont?

File: src/pages/Login/Login.jsx [hunk 3]
```diff
@@ -1,25 +1,46 @@
...
+  const [passwordValid, setPasswordValid] = useState(false);
+
+  const setEmailFieldValid = (validVal) => {
+    setEmailValid(validVal);
+  };
+  const setPasswordFieldValid = (validVal) => {
+    setPasswordValid(validVal);
+  };
+
+  const [valid, setValid] = useState(false);
```
Comment (myroslavsuprun) [inline src/pages/Login/Login.jsx:20]: Remove useState and useEffect

**Review by myroslavsuprun — CHANGES_REQUESTED** [review]

**Review by myroslavsuprun — CHANGES_REQUESTED** [review]
Refactor the code

File: src/pages/Login/Login.jsx (hunk 2 above)
Comment (bilenkomike) [reply src/pages/Login/Login.jsx:41]: your mouth

**Pull request closed without merging**